tool/jt.rb

```diff
@@ -750,6 +752,42 @@ def cextc(cext_dir, test_gem=false, *clang_opts)
     end
   end
 
+  def pr(*args)
+    out, _err   = raw_sh('git remote', capture: true)
+    remotes     = out.split
+    remote_urls = remotes.map do |r|
+      out, _err = raw_sh("git config --get remote.#{r}.url", capture: true)
+      [r, out.chomp!]
+    end
+
+    upstream = remote_urls.find { |r, u| u.include? 'graalvm/truffleruby' }.first
+    bb       = remote_urls.find { |r, u| u.include? 'ol-bitbucket' }.first
+
+    pr_number = args.first
+    if pr_number
+      github_pr_branch = "#{upstream}/pr/#{pr_number}"
+    else
+      fetch     = "+refs/pull/*/head:refs/remotes/#{upstream}/pr/*"
+      out, _err = raw_sh("git config --get-all remote.#{upstream}.fetch", capture: true)
+      raw_sh "git config --add remote.#{upstream}.fetch \"#{fetch}\"" unless out.include? fetch
+      raw_sh "git fetch #{upstream}"
```

Comment:
The intention is that we just maintain our branches on GitHub and use the tool to make a copy on bb to trigger the CI. I expect that in gated PR driven development model, we won't push named branches to bb manually any more, we'll just need a copy of the PR on bb. `guthub/pr/<number>` is very clear where to find the source, avoids name collisions, and does not require maintenance (can be just left there).